config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -83,6 +83,38 @@ periodics:
     testgrid-tab-name: periodic-e2e-release-0-4
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
     testgrid-num-failures-to-alert: "2"
+- name: periodic-cluster-api-provider-aws-e2e-conformance-release-0-4
+  decorate: true
+  interval: 3h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-aws
+      base_ref: release-0.4
+      path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191112-9f04410-1.16
+        command:
+          - "runner.sh"
+          - "./scripts/ci-e2e-conformance.sh"
```

Comment:
Should this be ci-conformance.sh?

Reply:
No, see https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1360/files#diff-a2c024158251d83fd27abc4413253616R1